src/app/find-help/accommodation-search/accommodation-search.component.ts

```diff
@@ -29,17 +31,20 @@ export class AccommodationSearchComponent {
     }
   }
 
-  search(searchCriteria: AccommodationQuery) {
+  search(searchCriteria?: AccommodationQuery, pagination?: Pageable) {
     this.loading = true;
 
-    // TODO specify the params; at least sorting is configurable from the UI
+    this.searchCriteria.capacity = searchCriteria?.capacity || this.searchCriteria.capacity;
+    this.searchCriteria.location = searchCriteria?.location || this.searchCriteria.location;
+    this.pagination = pagination || this.pagination;
```

Comment:
`||` is dangerous as `0` is often a valid value. Always use `??` for such fallback logic which only catches `null` and `undefined`.

```suggestion
    this.searchCriteria.capacity = searchCriteria?.capacity ?? this.searchCriteria.capacity;
    this.searchCriteria.location = searchCriteria?.location ?? this.searchCriteria.location;
    this.pagination = pagination ?? this.pagination;
```